### ISSUE TITLE
Cannot add Xref to States in PathVisio Java application

### ISSUE THREAD

**Issue by egonw** [PathVisio/pathvisio#66]
But this is supported in the GPML, AFAIK, and used in the FTO pathway:

![image](https://user-images.githubusercontent.com/26721/38161501-539ee6ba-34d0-11e8-98b4-5ff2d257b86d.png)

With the matching GPML:

```xml
<State GraphRef="bd803" TextLabel="C risk allele" GraphId="b2ba5">
  <Comment>C risk allele (rs1421085)</Comment>
  <Graphics RelX="1.5" RelY="1.0" Width="75.0" Height="15.0" ShapeType="Oval"/>
  <Xref Database="dbSNP" ID="rs1421085"/>
</State>
```

**Comment by egonw** [comment]
BTW, off-topic, but, @fehrhart, is this something we can use your new link sets for?

**Comment by DeniseSl22** [comment]
In the specific PW you mention, Freddy added this directly to the GPML. So it is not directly visible in the Pathway, nor is it transferred to the RDF.

**Comment by DeniseSl22** [comment]
I think @khanspers is using a special encoding for the States; I don't think there are any identifiers that can be used there right? If we could add an ID for a SNP, we could add mutations more clearly in a PW. Any other usecases?

**Comment by khanspers** [comment]
Right, at least for phosphorylation sites there is not an ID that could be used AFAIK.
